src/pages/home/report/ReportActionCompose.js

```diff
@@ -21,7 +21,7 @@ import compose from '../../../libs/compose';
 import PopoverMenu from '../../../components/PopoverMenu';
 import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
```

Comment:
We should use the hook equivalents for `withWindowDimensions` and `withLocalize` and remove the associated props.

Reply:
Same for:

- `withNetwork`
- `withKeyboardState`
- `withAnimatedRef`
- `Permissions` and the `betas` Onyx key
- `withNavigation`
- `withNavigationFocus`